src/Mono.Android/map.csv

```diff
@@ -98,26 +63,37 @@
 //1,Android.Views.WindowManagerTypes, LastSystemWindow, android/view/WindowManager$LayoutParams.LAST_SYSTEM_WINDOW, 2999
 //11,Android.Views.KeyEventFlags, Fallback, android/view/KeyEvent.FLAG_FALLBACK, 1024
 
+
+
+
+
+
+
+
+
+
+
+
 // - Automatically generated mappings (see tools/enumification-helpers how map.ext.csv is generated.)
 
-1,Org.XmlPull.V1.XmlPullParserNode,Cdsect,I:org/xmlpull/v1/XmlPullParser.CDSECT,5
```

Comment:
As per a previous comment, I don't understand why the first column here actually needed to change.